### PR TITLE
Data: Rest Api Specs - regenerating based on 9c7c83c441857e5173e62106408f510565ea1219

### PR DESCRIPTION
**Pull request by hc-github-team-tf-azure**
This PR is automatically generated based on the commit 9c7c83c441857e5173e62106408f510565ea1219

### CODE REVIEW

**Comment by hc-github-team-tf-azure** [comment]
No new resource IDs found.

File: data/Pandora.Definitions.ResourceManager/Automation/v2019_06_01/SoftwareUpdateConfiguration/Operation-sCreate.cs [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
+internal class sCreateOperation : Pandora.Definitions.Operations.PutOperation
 {
     public override Type? RequestObject() => typeof(SoftwareUpdateConfigurationModel);
 
     public override ResourceID? ResourceId() => new SoftwareUpdateConfigurationId();
 
     public override Type? ResponseObject() => typeof(SoftwareUpdateConfigurationModel);
 
-    public override Type? OptionsObject() => typeof(SoftwareUpdateConfigurationsCreateOperation.SoftwareUpdateConfigurationsCreateOptions);
+    public override Type? OptionsObject() => typeof(sCreateOperation.sCreateOptions);
```
Comment (tombuildsstuff) [inline data/Pandora.Definitions.ResourceManager/Automation/v2019_06_01/SoftwareUpdateConfiguration/Operation-sCreate.cs:23]: problem

File: data/Pandora.Definitions.ResourceManager/Automation/v2019_06_01/SoftwareUpdateConfiguration/Operation-sDelete.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 namespace Pandora.Definitions.ResourceManager.Automation.v2019_06_01.SoftwareUpdateConfiguration;
 
-internal class SoftwareUpdateConfigurationsDeleteOperation : Pandora.Definitions.Operations.DeleteOperation
+internal class sDeleteOperation : Pandora.Definitions.Operations.DeleteOperation
```
Comment (tombuildsstuff) [inline data/Pandora.Definitions.ResourceManager/Automation/v2019_06_01/SoftwareUpdateConfiguration/Operation-sDelete.cs:15]: problem

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]

**Comment by tombuildsstuff** [comment]
Requires regenerating once #3017 is merged

**Pull request closed without merging**